haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
```

Comment:
Why is there a `\` here?

Reply:
@alaviss Are escape characters not needed?

Reply:
Of course it's allowed, but why would you need it? And is there any copyright year for the second entry?